views/course_view.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.actions.act_window" id="open_academy_course_action">
+        <field name="name">Courses</field>
+        <field name="res_model">course</field>
+        <field name="view_mode">tree,kanban,form</field>
+    </record>
+
+    <record model="ir.ui.view" id="open_academy_course_view_action">
```

Comment:
This is not an action. The standard for naming views is <model_name>_view_<view_type> , where view_type is kanban, form, tree, search

```suggestion
    <record model="ir.ui.view" id="open_academy_course_view_tree">
```
See the [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action) please. And apply this to all records in this file please.